invisible_cities/cities/detsim.py

```diff
@@ -213,9 +213,10 @@ def detsim(*, files_in, file_out, event_range, print_mod, compression,
                                          create_pmt_waveforms,
                                          create_sipm_waveforms,
                                          get_bin_edges,
+                                         buffer_calculation,
                                          fl.branch("event_number"     ,
                                                    evtnum_collect.sink),
-                                         buffer_calculation),
+                                         fl.sink(lambda x: x)),
```

Comment:
What is the point of this? Is it to have a `/dev/null` sink? (One that just throws the data away?) 

If so, then simply don't branch off at the previous step!

Reply:
We could sink the pipe inside `calculate_and_save_buffers`, but I wanted to keep the `evtnum_collect` counter. I dont know how to `sink` the main pipe afterwards so I added this dummy `sink` that does nothing.

Reply:
I don't follow. At the moment you have

```
buffer_calculation ----> /dev/null
                  \
                    ---> pick event_number -> evtnum_collect.sink
```
Why can't you do the following instead?
```
buffer_calculation ---> pick event_number -> evtnum_collect.sink
```

Reply:
Yes, you are right. My bad because didn't really understand what `"event_number"` did inside a pipe.